tests/test_timeit_logger.py

```diff
@@ -0,0 +1,35 @@
+from __future__ import print_function
+import logging
+import unittest
+from mock import patch
+import profilehooks
+import sys
+import time
+
+
+class testProfilehooksTimeitLogger(unittest.TestCase):
```

Comment:
Class names ought to start with a capital letter.  Also, I think "profilehooks" is implied, so I'd shorten it to "TestTimeitLogger".